crates/bevy_render/src/pipelined_rendering.rs

```diff
@@ -77,11 +71,7 @@ impl Plugin for PipelinedRenderingPlugin {
         }
         app.insert_resource(MainThreadExecutor::new());
 
-        let mut sub_app = App::empty();
-        sub_app.add_stage(
-            RenderExtractStage::BeforeIoAfterRenderStart,
-            SystemStage::parallel(),
-        );
+        let sub_app = App::empty();
```

Comment:
just leaving the schedule as whatever App::empty sets it to. Probably need to set some things up here.